### ISSUE TITLE
Set environment vars per package

### ISSUE THREAD

**Issue by massimocode** [felixrieseberg/windows-build-tools#13]
Hi

So following https://github.com/felixrieseberg/windows-build-tools/issues/12 I noticed (from the Task Manager) that the build tools were installing. I let these install and also installed python manually from the version that was downloaded by running this tool.

However, I still can't install node-sass@3.4.2. I get the following error:
`MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "14.0", "4.0".`

Any ideas?

**Comment by massimocode** [comment]
I have found the solution discussed here: https://github.com/chjj/pty.js/issues/60

Basically I ran the following command:
`npm config set msvs_version 2015 --global`

I wanted to make a PR but I realised that there seems to be a script that does this once the downloads are complete. Considering the installation of Python was unsuccessful for me, that explains why this was not done automatically.

Feel free to close this issue if you think nothing further should be done... perhaps the environment variables for each thing should be done separately in case one fails and is installed manually by the user?
